package.json

```diff
@@ -197,11 +197,11 @@
     "JSONStream": "^1.0.3",
     "balena-config-json": "^4.1.0",
     "balena-device-init": "^5.0.2",
-    "balena-errors": "^4.4.1",
+    "balena-errors": "^4.7.1",
     "balena-image-manager": "^7.0.3",
     "balena-preload": "^10.3.1",
     "balena-release": "^3.0.0",
-    "balena-sdk": "^15.6.0",
+    "balena-sdk": "^15.20.0",
```

Comment:
The commit of title `npm-shrinkwrap: Bump balena-sdk to v15.20.0` is missing the following line in its body as explained in the [Contributing](https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md#semantic-versioning-commit-messages-and-the-changelog) document.
```
Update balena-sdk from 15.6.0 to 15.20.0
```
Btw, also as explained in the Contributing document, we should squash some commits before merging, as some commits are undoing what is done in earlier commits in this same PR. (I reckon that you are using many commits to make reviews easier.)

Reply:
@pdcastro yep as soon as it's ok I'll squash everything, keeping only 2 commits.  The module bump and the code changes.

Reply:
@mbalamat, re this commit:
```
packages: Bump balena-sdk and balena-errors

Update balena-sdk from 15.6.0 to 15.20.0
Update balena-errors from 4.4.1 to 4.7.1

Change-type: minor
Signed-off-by: Marios Balamatsias <mbalamatsias@gmail.com>
```
Somehow, versionbot thinks it is reason for a major CLI version bump, and proposes to release CLI v13:
* On the PR page, click the Checks tab, then navigate to Versionbot3 > VersionBot/generateversion
* Click on 13.0.0, which links to: https://github.com/balena-io/balena-cli/commit/b9fdcd8b767408160f88e7fa0838fd0e8021c255

I am not sure exactly why, but the Contributing document has some notes about specifying the desired version bump level between patch / minor / major. Also, the document clarifies that only packages listed in repo.yml should have a line in the commit body; `balena-sdk` is listed in `repo.yml`, while `balena-errors` is not. Try deleting the line regarding `balena-errors`, so the commit looks like:
```
packages: Bump balena-sdk and balena-errors

Update balena-sdk from 15.6.0 to 15.20.0

Change-type: minor
```